vta/include/vta/dpi/module.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef VTA_DPI_MODULE_H_
+#define VTA_DPI_MODULE_H_
+
+#include <tvm/runtime/module.h>
+#include <mutex>
+#include <queue>
+#include <condition_variable>
+#include <string>
+
+namespace vta {
+namespace dpi {
+
+class DPIModuleNode : public tvm::runtime::ModuleNode {
```

Comment:
do we use Doxygen comments on class definitions?

Reply:
Is there an example class documentation that you know I could use as a template?

Reply:
https://github.com/dmlc/tvm/blob/master/include/tvm/runtime/threading_backend.h#L36

Reply:
Done.

Reply:
Please also add documentation to all the functiond